manifests/enterprise/dashboard/package.pp

```diff
@@ -0,0 +1,33 @@
+class sensu::enterprise::dashboard::package {
+
+  if $caller_module_name != $module_name {
+    fail("Use of private class ${name} by ${caller_module_name}")
+  }
+
+  if $::sensu::install_repo and $::sensu::enterprise_dashboard {
+    $ensure = 'present'
+
```

Comment:
I think you need to add a dashboard version param for this or you'll be in a situation where you can't ever upgrade the dashboard through puppet on an existing server (and building a new one you aren't guaranteed to get what you had before with the same manifests)